src/components/controls.vue

```diff
@@ -98,68 +101,49 @@ export default {
 </script>
 
 <style lang="scss">
+.card-header {
```

Comment:
The `card-header` class is used by bootstrap and automatically applied to card header elements. I think it might be worth renaming this class to avoid conflicts and confusion with that bootstrap class. Maybe use `white-card-header` or `bg-white`?